language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ImplementMethodCodeAction.java

```diff
@@ -39,6 +39,12 @@ public class ImplementMethodCodeAction extends AbstractImplementMethodCodeAction
 
     public static final String NAME = "Implement Method";
 
+    @Override
+    public boolean validate(Diagnostic diagnostic, DiagBasedPositionDetails positionDetails, 
```

Comment:
We can skip overriding this right? It will pickup the validate() method of the parent class then.

Reply:
+1